packages/salesforcedx-vscode-core/src/commands/commands.ts

```diff
@@ -230,7 +231,7 @@ export abstract class SelectDirPath
   public async gather(): Promise<
     CancelResponse | ContinueResponse<{ outputdir: string }>
   > {
-    const rootPath = vscode.workspace.rootPath;
+    const rootPath = getRootWorkspacePath();
```

Comment:
per my question in my PR description, this uses ( through the util ) workspace.workspaceFolders[0].uri.path and not uri.fsPath.

Reply:
We shouldn't use `uri.path` since that breaks file paths in windows, it doesn't return proper file path separators based on platform. Let's change it so everything uses `uri.fsPath`. 

Here's more info on that if needed https://code.visualstudio.com/api/references/vscode-api#Uri